prisma/migrations/20211115182559_availability_issue/migration.sql

```diff
@@ -0,0 +1,21 @@
+
+insert into "Availability" ("userId", "startTime", "endTime", "days")
+select
+  id as "userId", 
+  CAST(CONCAT(CAST(("startTime" / 60) AS text), ':00') AS time) AT TIME ZONE "timeZone" AT TIME ZONE 'UTC',
+  CAST(CONCAT(CAST(("endTime" / 60) AS text), ':00') AS time) AT TIME ZONE "timeZone" AT TIME ZONE 'UTC',
+  ARRAY [0,1,2,3,4,5,6]
+from 
+  (
+    select 
+      users.id, 
+      users."startTime", 
+      users."endTime", 
+      users."timeZone",
+      count("Availability".id) as availability_count
+    from users 
+    left join "Availability" on "Availability"."userId" = users.id
+    where users."timeZone" != ''
```

Comment:
- There might be a hidden issue here as dayjs' timezones is not guaranteed to match Postgres built-in timezones & we don't _actually_ validate that the input is a timezone
- I'd ideally do something like `WHERE users.timeZone IS VALID TIMEZONE` here but I can't google my way on how to do that here